packages/lesswrong/themes/siteThemes/eaTheme.ts

```diff
@@ -244,24 +244,15 @@ export const eaForumTheme: SiteThemeSpecification = {
           }
         },
         Header: {
-          root: {
-            height: 90,
-            '@media (max-width: 959.95px) and (min-width: 600px)': {
-              height: 86, // I don't know why headroom shifts by 4 pixels, don't ask me
+          appBar: {
+            padding: '1px 20px',
+            [defaultTheme.breakpoints.down('sm')]: {
+              padding: '1px 11px',
             },
             [defaultTheme.breakpoints.down('xs')]: {
-              height: 77,
+              padding: '9px 11px',
             },
           },
-          appBar: {
-            padding: 11,
-            '@media (min-width: 960px)': {
-              paddingLeft: 20,
-              paddingRight: 20,
-              paddingTop: 13,
-              paddingBottom: 13,
-            }
-          },
```

Comment:
Kinda weird that this is here, I'm not sure if there's a good reason for it to be here...

Reply:
We no longer prefer to override settings in this way. It's legacy from Vulcan